web/src/views/MapProjection.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <v-container class="grey lighten-3 pa-0 ma-0" fill-height>
+    <div id="map" ref="mapElement" />
+  </v-container>
+</template>
+
+<style scoped>
+#map {
+    width: 500px;
+    height: 500px;
+    padding: 0;
+    margin: 0;
+}
+</style>
+
+<script lang="ts">
+
+import {
+  ref, defineComponent, onMounted, PropType, computed,
+} from '@vue/composition-api';
+import geo from 'geojs';
+import { getEmbeddings, getTileMetadata } from '../datasets';
+import type { DatasetEmbedding, DatasetID } from '../datasets';
+
+interface RootDatasetEmbedding {
+  id: null;
+  child_bounding_box: [number, number, number, number];
+  parent: null;
+  child: DatasetID;
+  [key:string]: any;
+}
+
+interface StackFrame {
+  embedding: DatasetEmbedding | RootDatasetEmbedding,
+  parent: {
+    scale: number;
+    offset: {
+      x: number;
+      y: number;
+    };
+  };
+  treeDepth: number;
+}
+
+export default defineComponent({
+  name: 'MapProjection',
+
+  setup() {
+    const mapElement = ref(null);
+
+    onMounted(async () => {
+
+      const embeddings: Array<DatasetEmbedding> = getEmbeddings(2);
+      const rootDatasetID: DatasetID = embeddings.find(e => embeddings.every(x => x.child !== e.parent)).parent;
```

Comment:
This could be `undefined` meaning: no root node. It could also pick the first root node of many. We ought to implement backend constraints to prevent either case.